sounds/generate-messages.sh

```diff
@@ -5,6 +5,20 @@ create_audio_file () {
     espeak-ng -s 140 -l en "$2" -w >(ffmpeg -i - -y -sample_fmt s16 -ar 8000 files/"$1".wav)
 }
 
+create_ding_dong_file () {
+    sample_rate=8000
+    duration=2  # Duration of each "ding" and "dong" sound in seconds
+    fade_in_duration=1  # Fade-in duration in seconds
+    fade_out_duration=1  # Fade-out duration in seconds
+    sox -n -r $sample_rate -e signed-integer -b 16 -c 1 files/_ding.wav synth $duration sine 240 fade h $fade_in_duration $duration $fade_out_duration
```

Comment:
please use a more natural sound instead of a pure sine wave. the old one was a nice touch, here are similar sounds with a CC license: https://freesound.org/search/?q=clock